lib/plugins/inspec-parallel/lib/inspec-parallel/list_reporter.rb

```diff
@@ -20,7 +20,7 @@ def example_passed(notification)
         @resource_formatter = InspecPlugins::Parallelism::ListReporter::ResourceFormatter::Base.formatter_for(resource_name, parameters: parameters, query: query)
       end
 
-      puts "-t " + resource_formatter.format_id_for_list(raw_id)
+      puts "-t " + resource_formatter.format_id_for_list(raw_id) + " --reporter json:<%= pid%>.json"
```

Comment:
Good catch. I'm not sure PID is the right thing - maybe something based on raw_id - but something is needed for reporter.

Reply:
You mean to say to use raw_id for the output file?